### ISSUE TITLE
Missing accessibility HTML content

### ISSUE THREAD

**Issue by KevinLane36** [Kmoran1512/Math-Battle#13]
Each page seems to be lacking proper headers or img alt text ect.
I would try putting it into https://wave.webaim.org/ and seeing the results.

**Comment by Kmoran1512** [comment]
Out of curiosity did you put it into the search bar on the website itself? That can have problems with css apis, so a lot of the styling won't come through and things won't be properly labeled, you might want to try the plugin from the same website, it works a lot better, for me that is showing each page is fine, but if you find any problems please let me know.
Thank you for the heads up about the pictures, I am working on those now.

Thank you!
